core/new-gui/src/app/workspace/service/workflow-graph/model/workflow-action.service.ts

```diff
@@ -1085,6 +1086,18 @@ export class WorkflowActionService {
     };
   }
 
+  public setTempWorkflow(workflow: Workflow): void {
+    this.tempWorkflow = workflow;
+  }
+
+  public resetTempWorkflow(): void {
+    this.tempWorkflow = <Workflow>{};
```

Comment:
can you make it undefined instead of empty workflow

Reply:
done